### ISSUE TITLE
Modify right panel to include what sites are already loaded

### ISSUE THREAD

**Issue by SRGDamia1** [ODM2/ODM2DataSharingPortal#585]
After loading just one site, the right panel is long enough to need a scroll bar.  If you load a second site, the new series get tacked on to the bottom of the list so it appears nothing has happened when you click the load button.

The GUI might need some re-arranging to more effectively use space.

![image](https://user-images.githubusercontent.com/5374350/163269231-c4c03e5d-386d-4402-bf2f-f5c78453d0b0.png)

**Comment by ptomasula** [comment]
I agree. I think the way the series selection subpanel is populated currently might not be the most logically. I think it's worth thinking through how to better use the real estate on the control panel. Maybe even grouping series by site under an accordion or something.
